PyRoute/wikistats.py

```diff
@@ -359,7 +359,7 @@ def text_area_long(self, f, area_type, area):
             nlist.append(self.get_count(nobles.nobles['Baronets'], 'Baronet', False))
             nlist.append(self.get_count(nobles.nobles['Barons'], 'Baron', False))
             nlist.append(self.get_count(nobles.nobles['Marquis'], 'Marquis', False))
-            nlist.append(self.get_count(nobles.nobles['Vicounts'], 'Viscount', False))
+            nlist.append(self.get_count(nobles.nobles['Viscounts'], 'Viscount', False))
```

Comment:
This is the only one I couldn't directly test, but it follows the same pattern as the changes in the Nobles object.